pkg/sslpolicy.go

```diff
@@ -13,10 +13,10 @@ import (
 // necessary to create our globally enforced SSL Policy.
 func NewSslPolicy(name string) compute.SslPolicy {
 	return compute.SslPolicy{
-		Description:   "Commercetools TLS policy: modern features and TLS 1.2 only.",
+		Description:   fmt.Printf("Commercetools TLS policy: %s features and above TLS %s only.", config.SslProfile(), config.TlsVersion()),
 		Name:          name,
-		Profile:       "MODERN",
-		MinTlsVersion: "TLS_1_2",
+		Profile:       config.SslProfile(),
```

Comment:
This func will be failed if ENV vars not set.

Reply:
In config.SslProfile I would check for valid value from ENV var, then default to something if ENV var is not set or invalid.

Reply:
Fixed by https://github.com/commercetools/gcp-ssl-policy-asserter/pull/3/commits/a112cf3f9dea2b9f4167d448f92a25e535e4ae10